website/docs/workloads/blender/blender.md

```diff
@@ -0,0 +1,57 @@
+# Blender Benchmark
+Blender Benchmark, a new platform to collect and display the results of hardware and software performance tests. With this benchmark we aim at an optimal comparison between system hardware and installations, and to assist developers to track performance during Blender development.
+
+The benchmark consists of two parts: a downloadable package which runs Blender and renders on several production files, and the Open Data portal on blender.org, where the results will be (optionally) uploaded.
```

Comment:
Do we do any uploading part? or you copied it from somewhere?

Reply:
No we don't. Sorry for the misleading info. will rewrite this doc.